### DISCUSSION TITLE
UPPAAL GUI is not fully displayed

### DISCUSSION THREAD

**Discussion by Lqs66** [UPPAALModelChecker/UPPAAL-Meta#270] [Q&A]
I encountered the problem shown below when using the UPPAAL GUI on my ubuntu 22.04 VM:

UPPAAL version : UPPAAL 5.0.0 (rev. 714BA9DB36F49691), 2023-06-21

![image](https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/62269820/a3c0a47e-6a88-4db7-a233-af10740ad276)

**Comment by mikucionisaau** [comment]
That's likely a graphics driver issue.

**Reply by Lqs66** [reply]
Thanks, how do I fix it?

**Reply by mikucionisaau** [reply]
Well, it's your graphics vendor.
You mention that it is "VM", so look for "java graphics [your vm name]". It could also be your host drivers -- it's difficult to say anything specific as I have no details what system you are using.

I use VirtualBox, so usually installing guest additions and setting the video memory to 128MB helps, also "3D acceleration" option often causes issues.

**Reply by Lqs66** [reply]
I tried typing java graphics lqs66.
![image](https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/62269820/4c39c865-103d-48ef-aa23-724ab857dfca)

**Reply by mikucionisaau** [reply]
No, no, search the internet :-)
What is the VM you are using? I don't think lqs66 is a VM.

**Reply by Lqs66** [reply]
Oh, I misinterpreted what you meant, thanks for the reply.
